### ISSUE TITLE
Clip path code broken in latest master

### ISSUE THREAD

**Issue by bigianb** [dc-js/dc.js#789]
Since updating to master, I see the following error when displaying a chart:
`Failed to execute query: '#1-clip' is not a valid selector.`
I'm pretty sure this is due to pr #733 

Digging a bit more, the issue is that the default anchor name starts with a number ... and DOM ids must start with a letter. Making the change below (i.e. changing '' in the last line to 'id') fixes the issue as the id now starts with a letter.
```
_chart.anchorName = function () {
        var a = _chart.anchor();
        if (a && a.id) {
            return a.id;
        }
        if (a && a.replace) {
            return a.replace('#', '');
        }
        return 'id' + _chart.chartID();
    };
```

**Comment by r4j4h** [comment]
Just updating this issue ticket that my PR is now good to go. :) :+1:

**Comment by Frozenlock** [comment]
Just wanted to chime in: I get the same error when trying any version after early December.

Without being familiar with the code, I tried to pinpoint the origin:
https://github.com/dc-js/dc.js/commit/da2d4eeb1abdd0c12f83d81e5480933e2a9feb45#commitcomment-9174465

**Comment by gordonwoodhull** [comment]
fixed by #800
